frontend/app/components/project-details.js

```diff
@@ -6,7 +6,11 @@ export default Component.extend({
 
   buildYearRange: Array.from({length: (2040 - 2018)}, (v, k) => k + 2018),
 
+  // noop
+  save() { return; },
```

Comment:
I know this is necessary, I just don't know why. Can you explain it to me?

Reply:
Ah, it's like self-documentation. I didn't understand where `save` was coming from - realized it was being passed through.

Additionally, it's easier to unit test this component when it's clear what the main event of this component (this action) is a function.

Happy to remove but I think it's best to think of components as self-contained islands!

Reply:
Word.